recipes/agg-regrid/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "agg-regrid" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "1ab034cd9c5363afcd2cf74073bf11ab16d9e3c81729433eb008fcb6e81be976" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/SciTools-incubator/iris-{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [osx]
```

Comment:
Is this just because Travis wasn't running or because it actually doesn't work on Mac?

Reply:
You can also try closing and reopening the PR to trigger travis again